backend/JiperBackend/Services/Context.cs

```diff
@@ -11,10 +11,12 @@ public Context(DbContextOptions<Context> options) : base(options)
         }
 
         public DbSet<User> Users { get; set; }
+        public DbSet<CourierCompany> CourierCompanies { get; set; }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<User>().ToTable("User");
+            modelBuilder.Entity<User>().ToTable("CourierCompany");
```

Comment:
should be modelBuilder.Entity<CourierCompany> i think

Reply:
It should, I'll change that.